content/en/tagging/assigning_tags.md

```diff
@@ -77,11 +77,48 @@ When installing the containerized Datadog Agent, set your host tags using the en
 DD_KUBERNETES_POD_LABELS_AS_TAGS='{"app":"kube_app","release":"helm_release"}'
 DD_DOCKER_LABELS_AS_TAGS='{"com.docker.compose.service":"service_name"}'
 ```
-When using the `DD_DOCKER_LABELS_AS_TAGS` variable within a Docker Swarm `docker-compose.yaml file`, remove the apostrophes, for example:
+When using the `DD_DOCKER_LABELS_AS_TAGS` variable within a Docker Swarm `docker-compose.yaml` file, remove the apostrophes, for example:
 
 ```shell
 DD_DOCKER_LABELS_AS_TAGS={"com.docker.compose.service":"service_name"}
 ```
+When adding labels to Docker containers, the placement of the `labels:` keyword inside the `docker-compose.yaml` file is very important. If the container needs to be labeled then place the `labels:` keyword **inside** the `services:` section **not** inside the `deploy:` section. Place the `labels:` keyword inside the `deploy:` section only when the service needs to be labeled. The Datadog agent will not have any labels to extract from the containers without this placement. Below is a sample, working `docker-compose.yaml` file that shows this. In the example below, the labels in the `myapplication:` section, `my.custom.label.project` and `my.custom.label.version` each have unique values. Using the `DD_DOCKER_LABELS_AS_TAGS` environment variable in the `datadog:` section will extract the labels and produce these two tags for the `myapplication` container:
+
+Inside the `myapplication` container the labels are: `my.custom.label.project` and `my.custom.label.version`
+
+After the agent extracts the labels from the container the tags will be:
+`projecttag:projectA`
+`versiontag:1`
+
+**Sample docker-compose.yaml:**
+```shell
```

Comment:
Should this be:

```
```yaml
```

Instead of:

```
```shell
```
?

Not sure where we source or syntax highlighting from? If it is from Linguist (what GitHub uses) it seems it would be worth changing this for proper highlighting? (looking at the formatting on the page itself once rendered looks 🆗 but not sure it is accurate?)

Reply:
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5590